documentation/docs/00-introduction.md

```diff
@@ -19,6 +19,38 @@ Building an app with all the modern best practices — code-splitting, offline s
 
 You don't need to know Svelte to understand the rest of this guide, but it will help. In short, it's a UI framework that compiles your components to highly optimized vanilla JavaScript. Read the [introductory blog post](https://svelte.dev/blog/svelte-3-rethinking-reactivity) and the [tutorial](https://svelte.dev/tutorial) to learn more.
 
+### Rendering on the web
+
+The core of SvelteKit provides a highly configurable rendering engine. This section describes some of the terms used when discussing that rendering engine. A reference for setting these options is provided later in the documentation.
+
+#### SSR
+
+Server-side rendering (SSR) is the generation of the page contents on the server. SSR is generally better for SEO. While some search engines can index content that is dynamically generated on the client-side it may take longer even in these cases. It also tends to improve perceived performance and makes your app accessible to users if JavaScript fails or is disabled (which happens [more often than you probably think](https://kryogenix.org/code/browser/everyonehasjs.html)).
```

Comment:
better ... requires something to be better than
you have not yet mentioned CSR, which you are trying to dis

Reply:
changed "better" to "preferred". not trying to dis any approach but just point out the pros and cons of each